[RM Ch08] Extreme-edge devices reference

joaoarodrigues opened the pull request.

walterkozlowski commented:
At RM Meeting pn 23 November 2022: discussion that we do not cover orchestration in RM; discussion to be continued post Nile

joaoarodrigues commented:
The main point here was to consider extreme-edge devices, its not about the orchestration

CsatariGergely commented on doc/ref_model/chapters/chapter08.rst:
```diff
@@ -659,6 +659,8 @@ operations and management complexity.
 At the edge, the infrastructure may further include ruggedised hardware for harsh environments and hardware with
 different form factors.
 
+Finally, there is a need to integrate, and to consider extreme-edge devices which can be part of the ecosystem, and offer programability and processing capabilities. Ultimately, can be considered for the end-to-end orchestration, and can be used to comply with extreme requirements for some of the use cases. 
```
Please wrap lines to 120 chars.

joaoarodrigues replied:
```suggestion
Finally, there is a need to integrate, and to consider extreme-edge devices which can be part of the ecosystem,
 and offer programability and processing capabilities. Ultimately, can be considered for the end-to-end 
 orchestration, and can be used to comply with extreme requirements for some of the use cases. 
```

jbhartley commented:
Suggest:

1. We include content into prior paragraph
2. Clarify wording

Suggest:

At the edge, the infrastructure may further include ruggedised hardware for harsh environments and hardware with
different form factors. With the evolution of IoT and ubiquitous connectivety (inclusive of personal devices) to consider extreme-edge devices as part of the ecosystem.  This will require need to integrate with and offer programability and processing capabilities for these devices. The end-to-end orchestration will need to support the extreme edge use cases.

pgoyal01 commented on doc/ref_model/chapters/chapter08.rst:
```diff
@@ -659,6 +659,10 @@ operations and management complexity.
 At the edge, the infrastructure may further include ruggedised hardware for harsh environments and hardware with
 different form factors.
 
+Finally, there is a need to integrate, and to consider extreme-edge devices which can be part of the ecosystem,
```
As discussed in the RM meeting of 2022-12-07, starting the sentence with "Finally" doesn't follow the previous sentences. If this suggestion is accepted, then lines 663 and 664 should be deleted. Orchestration is not the topic of this section and so is being removed.

If orchestration must be mentioned, then it should be a separate paragraph that applies to the content of this section.
```suggestion
Additionally, the extreme edge may consist of programmable devices as part of the
Telco Edge Cloud ecosystem for use in meeting some extremely low latency requirements,
```

steelescotr commented:
@walterkozlowski - would you be able to finalize this PR so we can be sure it's in the Nile Release?

joaoarodrigues commented:
> Suggest:
> 
> 1. We include content into prior paragraph
> 2. Clarify wording
> 
> Suggest:
> 
> At the edge, the infrastructure may further include ruggedised hardware for harsh environments and hardware with different form factors. With the evolution of IoT and ubiquitous connectivety (inclusive of personal devices) to consider extreme-edge devices as part of the ecosystem. This will require need to integrate with and offer programability and processing capabilities for these devices. The end-to-end orchestration will need to support the extreme edge use cases.

@jbhartley Thank you for the proposal. I've updated it, and corrected a couple of typo's 😊

joaoarodrigues commented:
> @walterkozlowski - would you be able to finalize this PR so we can be sure it's in the Nile Release?
